github/challenge_processing_script.py

```diff
@@ -35,6 +35,10 @@
 
 if __name__ == "__main__":
 
+    if GITHUB_CONTEXT["event"]["head_commit"]["message"].startswith("evalai_bot"):
+        print("Sync from Evalai")
+        sys.exit(0)
+
```

Comment:
@savish28 can you explain why we need this check ?

Reply:
@Ram81, this is prevent syncing back changes to EvalAi which came through evalai. So the the comnits we make start with this.